assets/scss/2-layout/1-header.scss

```diff
@@ -1,5 +1,11 @@
 header {
   padding: 20px 0px;
+  position: fixed;
```

Comment:
Have you had the chance to look into `sticky` option instead?

Reply:
Yes, but its compatibility is very poor https://caniuse.com/?search=sticky. Should I still use it?

Reply:
Great point. 👍 
In this case we are not concerned with maximum application portability, just exploration of possible feature options, you are free to stick to whichever solution you feel best. 👍

Reply:
I understood your thought. In this case I can live in ideal world where everyone is using latest chrome version 🥇 haha